src/main.cpp

```diff
@@ -12,6 +13,8 @@ int main(int argc, char** argv)
     MainWindow* mainWindow = nullptr;
     refBuilder->get_widget_derived("main_window", mainWindow);
     pMainWindow.reset(mainWindow);
+    
+    TrayIcon trayIcon(mainWindow, refBuilder);
```

Comment:
auto const trayIcon = TrayIcon{mainWindow, refBuilder} to have a convention consistency.